source/query-bar.txt

```diff
@@ -82,8 +79,9 @@ To set projection, click :guilabel:`Options` and enter in the
 projection document:
 
 - To include fields, specify the field name and set to ``1`` in the
-  project document. For example:
+  project document:
 
+  .. example::
```

Comment:
Fixed.  Also added " ... as in following example" text in place of the Example directive.